### PR TITLE
Add gsassetrules

### PR DESCRIPTION
**Pull request by joyceseng**

### CODE REVIEW

File: content/overview/get-started/gs-asset-rules.md [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+3. [Map tokens](#map-tokens-#3): Map the tokens to values.
+
+4. [Configure assets](#configure assets-#4): Assign tokens values to asset fields.
+
+5. [Preview assets](#preview-assets-#5): Display a preview of the assets that will be created by this rule. 
+
+For more information about asset rules, see [Asset rules](xref:ccAssetRules).
+
+
+- [Asset rules](xref:ccAssetRules)
```
Comment (afloresOSI) [inline content/overview/get-started/gs-asset-rules.md:42]:
```suggestion

```
Remove bulleted list.

File: content/overview/get-started/gs-asset-rules.md [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+
+1. In the left pane, select  **Data Management>Asset Rules**.
+
+1. From the **Namespace** dropdown list, ensure the correct namespace is selected.
+
+1. In the toolbar, select **Add a Rule**.
+
+1. In the **Create New Asset Rule** window, enter the following information.
+
+   - `Name` &ndash; **MyData.AssetRule**
```
Comment (afloresOSI) [inline content/overview/get-started/gs-asset-rules.md:58]: Needs blank lines after each bullet.

File: content/overview/get-started/gs-asset-rules.md [hunk 3]
```diff
@@ -0,0 +1,145 @@
...
+
+   The window should display the **MyData.NorthAmerica.SLTC.PumpA** and **MyData.NorthAmerica.SLTC.PumpB** streams. 
+
+1. Select the **MyData.NorthAmerica.SLTC.PumpA** stream and select **Add**.
+
+## <a name="gsAssetRules-extract-tokens"></a> Extract tokens
+
+Assemble token values so that you can select the information you want to extract from your streams. 
+
+1. Initially, **MyData** should be highlighted. Use the slider to select the `.` character, so that **MyData.** is highlighted.
```
Comment (afloresOSI) [inline content/overview/get-started/gs-asset-rules.md:74]: Needs blank lines between each numbered item.

File: content/overview/get-started/gs-asset-rules.md [hunk 4]
```diff
@@ -0,0 +1,145 @@
...
+1. Next, **SLTC** is selected. In this case, we know our naming pattern enforces a 4-letter abbreviation for the site, so in **3. Match:** select **the next 4 letters**. This helps ensure we do not match any streams that we do not expect. 
+1. Name this token **Site** and select **Capture**.
+1. Now, **PumpA** is selected. Because we want to distinguish pump streams from other streams, use the slider to select only **Pump**, and select **the next 4 characters of any type**. 
+1. Name this token **Equipment** and select **Capture**.
+1. Finally, just **A** is selected. Use the option **letters until the end of stream name**, and name this token **Id**. 
+1. Select **Capture**.
+
+**Note**: Alternatively, you can bring in tokens by using the **Stream Metadata** button. 
+
+
```
Comment (afloresOSI) [inline content/overview/get-started/gs-asset-rules.md:90]: There is an extra line here.

File: content/overview/get-started/gs-asset-rules.md [hunk 5]
```diff
@@ -0,0 +1,145 @@
...
+1. Select **Next**.
+
+## <a name="gsAssetRules-configure-assets"></a> Configure assets
+
+Assign tokens values to asset fields. 
+
+**Note**: You can enter  the `{` character to show a list of available tokens. 
+
+1. In the **Configure Asset** panel, enter the following information.
+   - Id &ndash; **{Id}**
```
Comment (afloresOSI) [inline content/overview/get-started/gs-asset-rules.md:131]: Need a blank line after each bullet and number.

File: content/overview/get-started/gs-asset-rules.md [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+   - Name &ndash; **{Equipment}{Id}**
+   - Description &ndash; **{Region} {Site} Pump State**
+1. Select  **Next**.
+
+## <a name="gsAssetRules-preview-assets"></a> Preview assets
+
+Display a preview of the assets that will be created by this rule. 
+
+1. Use the view selection at upper right to toggle between list and table view of the assets. The **Asset Preview** panel at left shows the rule details and statistics.
```
Comment (afloresOSI) [inline content/overview/get-started/gs-asset-rules.md:140]: Needs a blank line.

**Review by afloresOSI — CHANGES_REQUESTED** [review]

**Comment by joyceseng** [comment]
Are some of these the same changes you requested in the PR I closed?? Just asking for future reference, as I thought I had put them in ☹

File: content/overview/get-started/gs-asset-rules.md (hunk 2 above)
Comment (joyceseng) [reply content/overview/get-started/gs-asset-rules.md:53]: I think I fixed the things you caught!

**Review by osisoft-mbishop — APPROVED** [review]
@joyceseng , you indeed added breaks between bullets. Looks good for merge.

**Comment by joyceseng** [comment]
OMG! I am so relieved!

**Comment by joyceseng** [comment]
Thank you, Mark!

**Pull request merged**